neo/Prompt/Utils.py

```diff
@@ -334,3 +339,33 @@ def gather_param(index, param_type, do_continue=True):
             return gather_param(index, param_type, do_continue)
 
     return None, True
+
+
+def get_token(wallet: 'Wallet', token_str: str) -> 'NEP5Token.NEP5Token':
+    """
+    Try to get a NEP-5 token based on the symbol or script_hash
+
+    Args:
+        wallet: wallet instance
+        token_str: symbol or script_hash (accepts script hash with or without 0x prefix)
+    Raises:
+        ValueError: if token is not found
+
+    Returns:
+        NEP5Token instance if found.
+    """
+    if token_str.startswith('0x'):
+        token_str = token_str[2:]
+
+    token = None
+    for t in wallet.GetTokens().values():
+        if token_str == t.symbol:
```

Comment:
Maybe `if token_str in [t.symbol, t.ScriptHash.ToString()]:` is a bit more elegant

Reply:
Changed!